tests/providers/qdrant/operators/test_qdrant.py

```diff
@@ -0,0 +1,70 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from datetime import datetime
+from unittest.mock import patch
+
+import pytest
+
+from airflow.models import DAG
+from airflow.providers.qdrant.operators.qdrant import QdrantIngestOperator
+
+
+@pytest.fixture
+def dummy_dag():
```

Comment:
You could use the available [`dag_maker` fixture](https://github.com/apache/airflow/blob/8914e49551d8ae5ece7418950b011c1f338b4634/tests/conftest.py#L611) here if you'd like.

Reply:
https://github.com/apache/airflow/pull/36805/commits/86c50b2e7f362e75b58a461ef5d3aa74e4ce8ae7